package.json

```diff
@@ -35,7 +35,7 @@
   "dependencies": {
     "@braintree/sanitize-url": "^7.0.1",
     "@companieshouse/api-sdk-node": "^2.0.160",
-    "@companieshouse/ch-node-utils": "^1.3.8",
+    "@companieshouse/ch-node-utils": "file:..~/dev/ch-node-utils",
```

Comment:
This will be changed once/if https://github.com/companieshouse/ch-node-utils/compare/tferns-IDVA6-1468-locale-changes?expand=1 is merged.

Reply:
https://github.com/companieshouse/your-companies-web/pull/175#:~:text=IDVA6%2D1468%3A%20Update%20ch%2Dnode%2Dutils